src/pharmpy/tools/external/nlmixr/run.py

```diff
@@ -21,32 +21,15 @@
     write_csv,
 )
 from pharmpy.tools import fit
-from pharmpy.workflows import default_tool_database
+from pharmpy.workflows import ModelEntry, default_tool_database
 from pharmpy.workflows.log import Log
 from pharmpy.workflows.results import ModelfitResults
 
 
-def execute_model(model: pharmpy.model.Model, db, evaluate=False, path=None) -> pharmpy.model.Model:
-    """
-    Executes a model using nlmixr2 estimation.
-
-    Parameters
-    ----------
-    model : pharmpy.model.Model
-        An pharmpy model object.
-    db : database for storing model results
-        Name of folder in home directory to store resulting files in.
-    evaluate : bool
-        Set to True to evaluate model instead of estimating
-    path : str
-        Set path of model files
-
-    Returns
-    -------
-    model : pharmpy.model.Model
-        Model with accompanied results.
+def execute_model(model_entry, db, evaluate=False, path=None):
+    assert isinstance(model_entry, ModelEntry)
+    model = model_entry.model
 
-    """
```

Comment:
Was the docstring removed on purpose?

Reply:
It was removed on purpose as the corresponding function in NONMEM has no docstring and the function is not intended to be used by a user. The same was done for the docstring in `pharmpy.tools.external.rxode/run.py`

Reply:
Ok, makes sense. This falls into some future documentation for how to support a new external tool.